### PR TITLE
feat-card

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (67)

### CODE REVIEW

File: comp-layouts/card.html [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <script src="https://kit.fontawesome.com/c4d0bd5372.js" crossorigin="anonymous"></script>
+     <title>cards</title>
+
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+     <link rel="stylesheet" href="/stylesheets/main.css" />
+     <link rel="stylesheet" href="/stylesheets/components/card.css" />
+     <link rel="stylesheet" href="/Stylesheets/abstract/color.css">
+     <title>Card</title>
```
Comment (shuntzhere) [inline comp-layouts/card.html:14]: Two titles here, also you import you stylesheets to the main css

File: comp-layouts/card.html [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+               <h2 class="txt con">NIKE FREE</h2>
+               <h3 class="txt con price">$55.07 <h5 class="dismiss">$65.45</h5>
+                    <span class="fa fa-star checked"></span>
+                   
+<span class="fa fa-star checked"></span>
+<span class="fa fa-star checked"></span>
+<span class="fa fa-star"></span>
+<span class="fa fa-star"></span>
+
+               </h3>
```
Comment (shuntzhere) [inline comp-layouts/card.html:34]: Please indent the code properly.

File: comp-layouts/card.html [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+                    <span class="fa fa-star checked"></span>
+                   
+<span class="fa fa-star checked"></span>
+<span class="fa fa-star checked"></span>
+<span class="fa fa-star"></span>
+<span class="fa fa-star"></span>
+               </h3>
+               <P class="txt con"> Color : Red</P>
+               <P class="txt con">  Size  : Large</P>
+               <i class="fa-thin fa-cart-shopping"></i>
```
Comment (shuntzhere) [inline comp-layouts/card.html:64]: Give con a more meaningful name.

File: stylesheets/base/reset.css [hunk 4]
```diff
@@ -1,3 +1,62 @@
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
```
Comment (shuntzhere) [inline stylesheets/base/reset.css:10]: Ye can move this above in *

File: stylesheets/components/card.css [hunk 5]
```diff
@@ -0,0 +1,131 @@
+@import url(/stylesheets/abstract/variables.css);
+@import url(/stylesheets/abstract/color.css);
+@import url(/Stylesheets/main.css);
+
```
Comment (shuntzhere) [inline stylesheets/components/card.css:4]: Import stylesheets either on Html or Css, don't do both. Also import all sheets in main.css and use that everywhere.

File: stylesheets/components/card.css [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+  width: 15rem;
+  background: var(--light-grey);
+  margin-left: 1rem;
+}
+.cardbox {
+  box-shadow: 7px 12px var(--dark-grey);
+}
+.card-only {
+  height: 11.5rem;
+  box-shadow: 20, 20 var(--bg-blue);
```
Comment (shuntzhere) [inline stylesheets/components/card.css:28]: This seems confusing, check this once.
Also, the whole point of box-shadow is to not make it completely visible. Go lighter.

File: stylesheets/components/card.css [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+
+.card_image2 {
+  height: 10rem;
+  width: 10rem;
+  background-size: cover;
+}
+
+.card-1 {
+  background-image: url("https://images.unsplash.com/photo-1542291026-7eec264c27ff?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxzZWFyY2h8MXx8c2hvZXxlbnwwfHwwfHw%3D&w=1000&q=80");
+}
```
Comment (shuntzhere) [inline stylesheets/components/card.css:46]: You can use this in HTML itself.

**Review by shuntzhere — CHANGES_REQUESTED** [review]
Indent the code for better readability. Also suggested some changes to be made.

File: comp-layouts/card.html (hunk 1 above)
Comment (iamsaurav28) [reply comp-layouts/card.html:14]: yes i did changes

File: comp-layouts/card.html (hunk 2 above)
Comment (iamsaurav28) [reply comp-layouts/card.html:34]: oh ya done

File: comp-layouts/card.html (hunk 3 above)
Comment (iamsaurav28) [reply comp-layouts/card.html:64]: done

**Review by therajatg — APPROVED** [review]
Do the changes suggested by shuntzhere. I don't find anything else out of place.

File: stylesheets/base/reset.css (hunk 4 above)
Comment (iamsaurav28) [reply stylesheets/base/reset.css:10]: okay done

File: stylesheets/components/card.css (hunk 5 above)
Comment (iamsaurav28) [reply stylesheets/components/card.css:4]: okay done

File: stylesheets/components/card.css (hunk 6 above)
Comment (iamsaurav28) [reply stylesheets/components/card.css:28]: ya i forgot to delete it. done

File: stylesheets/components/card.css (hunk 7 above)
Comment (iamsaurav28) [reply stylesheets/components/card.css:46]: ya but i think its ll be ok if i did here also.

Comment (shuntzhere) [reply stylesheets/components/card.css:46]: Definitely, it's okay, although it's good practice to put images in Html itself and change its styles on CSS if required.

**Review by shuntzhere — APPROVED** [review]
Looking good.

File: stylesheets/components/card.css (hunk 7 above)
Comment (iamsaurav28) [reply stylesheets/components/card.css:46]: okay i willl do it . ty for good observation

**Pull request merged**